docs/guides/serverless-api-with-nextjs.md

```diff
@@ -0,0 +1,515 @@
+# Nitric To-Dos
+
+In this tutorial, you'll create a [Next.js](https://nextjs.org/) application that connects to a task list [Nitric API](https://nitric.io/docs/apis) and a task list [Nitric Collection](https://nitric.io/docs/collections). We will also be using [Tailwind CSS](https://tailwindcss.com/) for styling.
```

Comment:
this sounds odd, I much prefer the wording in the overview

Reply:
Removed it, it was duplicating the overview section